lib/index.js

```diff
@@ -116,6 +123,8 @@ function getBackgroundImageSize(decl, existingBackgroundSize, detectImageSize, r
           prop: 'background-size',
           value: `${size.width}px ${size.height}px`,
         });
+      }).catch((err) => {
+        warn(err);
```

Comment:
Unfortunately not, I think it might return another resolved promise that doesn't pass `existingBackgroundSize`, so we end up with:

     @media (min-device-pixel-ratio: 1.5), (min-resolution: 144dpi), (min-resolution: 1.5dppx) {
       .dog {
         background-image: url('./test/fixtures/images/dog@2x.jpg');
    +    undefined: undefined;
       }
     }